db/seeds.rb

```diff
@@ -5,3 +5,5 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+
+FactoryBot.create :full_project
```

Comment:
Do you want to have this in here and linked to this PR?

Reply:
I needed some data to test the rake task. Figured I'd leave this in to spark the discussion about how we feel about building up some seed data. How do you feel about it?

Reply:
seed data sounds good, fancy tackling https://github.com/zooniverse/Panoptes/issues/2708 as well.

Reply:
Happy to. Were you wanting to run these on production too then, or is that also just an issue on dev setups?

Reply:
No need on production. The seed tasks should be for smoke testing dev / new installs. Ala the configure scripts for setting up yamls / jwt keys, etc.

Reply:
:+1: